### PR TITLE
add username and logout function to navbar

### PR DESCRIPTION
**Pull request by vlee-harmonicinc**
It still using the ng-controller of the original page that called navbar.html, so it have to include some function of it.

### CODE REVIEW

File: app/js/index.js [hunk 1]
```diff
@@ -110,11 +111,11 @@ angular.module('index-app', ['firebase'])
 							team: ['null']
 						});
 					}
+					$scope.username = usersArray.$getRecord(user.uid).name;
```
Comment (RonaldShum) [inline app/js/index.js]: Please do $scope.username = user.email; 

Bugs may occur when firebase is still setting up a new user profile and usersArray.$getRecord(user.uid).name is not there yet.

I will fix this and change it back to user name in next pull request! Thanks!

Comment (vlee-harmonicinc) [reply app/js/index.js]: sorry I saw that comment after I pull request second time.
I changed username to email then.

**Comment by RonaldShum** [comment]
navbar.html should be using the controller of that page(ie index controller/profile controller etc)

I don't think including multiple controllers in one page is possible since firebase can only be initialized in one of the controller in the page.

File: app/js/index.js (hunk 1 above)
Comment (RonaldShum) [reply app/js/index.js]: You don't need to submit another pull request. 
Changing code -> commit -> push again
Pull request will update automatically.

**Comment by vlee-harmonicinc** [comment]
@RonaldShum 
Yeah I found this problem so I changed navbar.js to just template.js for reference.

File: app/js/index.js (hunk 1 above)
Comment (vlee-harmonicinc) [reply app/js/index.js]: OK, I pushed.

**Comment by htleeab** [comment]
Since I cannot start my work (team leader) without event and it seem that some basic function is not assigned to anyone, I try to do the create event part.
I just do the basic function. There are still some other function of admin to be done, such as random (smart?) generator...
I move the storage of event under 'event' directory rather than the root of firebase (, which is the original storage method of peter) so that it will be easier to search through event and neat. Member pages and team leader page have to be move to 'event' directory now.
This is simple, just add 'event/' in front of the ref. path.

**Comment by vlee-harmonicinc** [comment]
Since I did not sure how to start to do team leader without creating event and it seem that some basic function is not assigned to anyone, I try to do the create event part.
I just do the basic functions. There are still some other function of admin to be done, such as random (smart?) generator. 
I move the storage of event under 'event' directory rather than the root of firebase (, which is the original storage method of peter) so that it will be easier to search through event and neat. Member pages and team leader page have to be move to 'event' directory now.
This is simple, just add 'event/' in front of the ref. path.

**Pull request merged**